### DISCUSSION TITLE
Missed member: hi-hyein

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#159] [General]
혜인님이 어제 건강상의 이유로 병가를 내시고 이 모임도 지속할 수 없다고 얘기해 주셔서
이번 모임 부터 못하게 되었습니다.

@hi-hyein 
부디 건강하세요. 좋은 모습으로 또 볼 날을 기대하겠습니다.

**Comment by Joseph-Cha** [comment]
> 혜인님이 어제 건강상의 이유로 병가를 내시고 이 모임도 지속할 수 없다고 얘기해 주셔서
> 
> 이번 모임 부터 못하게 되었습니다.
> 
> 
> 
> @hi-hyein 
> 
> 부디 건강하세요. 좋은 모습으로 또 볼 날을 기대하겠습니다.

😭 혜인님 건강 잘 챙기시고 좋은 모습으로 뵐 수 있기를 바랍니다 ㅠ

**Comment by momoci99** [comment]
혜인님 얼른 회복하시고 다시 뵐 날을 기다리고 있겠습니다 ㅜㅜㅜㅜㅜ
